conf/local-config.json

```diff
@@ -32,9 +32,10 @@
   "site_key_expires_after_seconds": 90000,
   "salt_snapshot_activates_in_seconds": 180,
   "salt_snapshot_expires_after_seconds": 180000,
-  "is_auth_disabled": true,
-  "oauth2_callback_url": "http://localhost:8089/oauth2-callback",
-  "github_client_id": "generate client id",
-  "github_client_secret": "generate client secret",
-  "enable_keysets": true
-}
+  "is_auth_disabled": false,
+  "enable_keysets": true,
+  "okta_client_id": "0oa1og75xc1tZZXEj1d8",
+  "okta_client_secret": "QhKBA9o-owxtrRWwg9LKZ7AktQK3-5L-igVQrlTalB0GoQYS2WGxXEM1M8ORmHHa",
```

Comment:
Are we supposed to check this into the repo? 🤔

Reply:
That is invalid, used for AbuTest app. Removed anyway